src/platform_impl/linux/wayland/event_loop.rs

```diff
@@ -623,6 +624,7 @@ impl<T: 'static> EventLoop<T> {
             }
         }
 
+        callback(Event::Suspended, &self.window_target, &mut control_flow);
```

Comment:
Missing X11. 

Also, did you check windows, macos, and ios's backends?

Reply:
This was just for testing locally. I'll submit a PR to winit master once I finish the event loop. Currently trying to preinstall parts of flutter-rs locally to accelerate the compile/run cycle. Due to all the security restrictions event on a rooted phone it seems to not be very easy. Maybe I'm just not doing things the way their supposed to be done, don't really know.